app/_hub/kong-inc/rate-limiting-advanced/index.md

```diff
@@ -285,6 +277,26 @@ params:
         header of denied requests (status = `429`) in order to prevent all the clients
         from coming back at the same time. The lower bound of the jitter is `0`; in this case,
         the `Retry-After` header is equal to the `RateLimit-Reset` header.
+    - name: enforce_consumer_groups
+      required: false
+      default: false
+      value_in_examples: true
+      datatype: boolean
+      description: |
+        Set to `true` to enable `consumer_groups`, which allows the settings
+        from one of the allowed consumer groups to override the given plugin
+        configuration.
+    - name: consumer_groups
```

Comment:
Can we add a note here (something like)
Flipping `enforce_consumer_groups` from `true` to `false` will disable the group override, but will not clear the list of `consumer_groups` provided here. You can flip `enforce_consumer_groups` to true to re-enforce the groups.